dts/arm/st/l0/stm32l071.dtsi

```diff
@@ -137,6 +137,24 @@
 			label = "UART_1";
 		};
 
+		usart4: serial@40004c00 {
+			compatible = "st,stm32-usart", "st,stm32-uart";
+			reg = <0x40004c00 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00080000>;
+			interrupts = <14 0>;
+			status = "disabled";
+			label = "UART_4";
+		};
+
+		usart5: serial@40005000 {
+			compatible = "st,stm32-usart", "st,stm32-uart";
+			reg = <0x40005000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00010000>;
```

Comment:
clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00**10**0000>;

Reply:
yeah, I missed it. Fixed now, thanks.